.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -118,110 +113,78 @@ jobs:
         set -x
         df -h
         for delete in /usr/share/dotnet /usr/local/lib/android /opt/ghc /opt/hostedtoolcache/CodeQL; do
-          rm -rf "${delete:?}"
+          rm -rf "/mnt${delete:?}"
         done
         df -h
 
-    - name: Set MANUAL_URL
-      id: set-manual-url
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: ${{ steps.split.outputs.branch }}
+
+    - uses: ./.github/actions/job-preamble
+      with:
+        gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
+
+    - name: Build updater
       run: |
-        if [ "${{ github.event_name }}" == "schedule" ]; then
-          echo "MANUAL_URL=https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
-        else
-          echo "MANUAL_URL=https://raw.githubusercontent.com/stackrox/stackrox/${{ github.event.pull_request.head.sha }}/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
-        fi
+        echo "Building updater for version ${{ steps.split.outputs.version }} on branch ${{ steps.split.outputs.branch }}..."
+        make tag
+        make -C scanner bin/updater
 
-    - uses: ./.github/actions/download-artifact-with-retry
+    - name: Upload Updater Artifact
+      uses: actions/upload-artifact@v4
       with:
-        name: updater
-        path: /usr/local/bin
+        name: updater-${{ steps.split.outputs.version }}
+        path: scanner/bin/updater
+        if-no-files-found: error
```

Comment:
Why are we creating an artifact for the built updater? Not seeing that used in this workflow.